packages/core/src/baseclient.ts

```diff
@@ -434,7 +497,15 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
           const isInternalException =
             hint && hint.data && (hint.data as { [key: string]: unknown }).__sentry__ === true;
           // We skip beforeSend in case of transactions
+
```

Comment:
While the new line is fine by itself, it created a strange gap to the comment above, which comes right after some unrelated (to the comment) const definition.

Reply:
Fixed, missed it.